lab-jonathan/index.js

```diff
@@ -0,0 +1,23 @@
+let hex = require('./lib/readHex.js');
+
+hex.read(function(data) {
+  for(var i = 0; i < data.length; i++) {
```

Comment:
Good stuff here. I'd love to see you using a `forEach()` here, and avoiding having to write out the for loop.